app/models/support/requests/content_publisher_feedback_request.rb

```diff
@@ -0,0 +1,32 @@
+require 'active_support/core_ext'
+
+module Support
+  module Requests
+    class ContentPublisherFeedbackRequest < Request
+      attr_accessor :feedback_type, :feedback_details, :impact_on_work
+
+      validates_presence_of :feedback_type
+
+      def initialize(attrs = {})
+        super(attrs)
+      end
+
+      def self.label
+        "Provide feedback on content publisher"
+      end
+
+      def self.description
+        "Suggest changes to features within the content publisher."
+      end
+
+      def feedback_type_options
+        [
+          ["an accessibility issue or a usability issue", "accessibility_or_usability"],
+          ["something working differently than you expected", "working_unexpectedly"],
+          ["a feature that would help you with your work", "helpful_feature"],
+          ["something else", "other"],
```

Comment:
Do we need these values to be snake case? - It looks like it produces a somewhat hard to read output in the email: https://github.com/alphagov/support/pull/521/files#diff-4f144761556ba2db8112558bccb3d2ecR19 and I assume we just need these to be strings?

Reply:
Done. Snake case removed.  I'll make a note more broadly on the repo to suggest we don't need this in the other forms.